sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -166,6 +217,45 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     });
   }
 
+  /**
+   * Creates a ministry notification for the saved education
+   * program offering as a part of the same transaction.
+   * @param educationProgramOffering information used to create the program offering.
+   * @param programOffering program offering to be saved.
+   * @param entityManager entity manager to be part of the transaction.
+   */
+  private async saveEducationProgramOfferingNotification(
+    educationProgramOffering: Pick<OfferingValidationModel, "offeringName"> &
+      Pick<OfferingValidationModel["programContext"], "name">,
+    programOffering: Pick<EducationProgramOffering, "offeringStatus"> & {
+      institutionLocation: Pick<
+        EducationProgramOffering["institutionLocation"],
+        "id" | "name"
+      >;
+    },
```

Comment:
The parameters are not improving the readability or maintainability.
Can you please declare them individually?
Because it is a private method, in this I would recommend even directly accepting the `validatedOffering` which would simplify the consumer code also.